torchaudio/datasets/librispeech.py

```diff
@@ -43,20 +43,20 @@ def download_librispeech(root, url):
     extract_archive(archive)
 
 
-def load_librispeech_item(
-    fileid: str, path: str, ext_audio: str, ext_txt: str
-) -> Tuple[Tensor, int, str, int, int, int]:
+def _get_librispeech_metadata(
+    fileid: str, root: str, url: str, ext_audio: str, ext_txt: str
+) -> Tuple[str, int, str, int, int, int]:
     speaker_id, chapter_id, utterance_id = fileid.split("-")
 
-    # Load audio
+    # Get audio path and sample rate
     fileid_audio = f"{speaker_id}-{chapter_id}-{utterance_id}"
-    file_audio = fileid_audio + ext_audio
-    file_audio = os.path.join(path, speaker_id, chapter_id, file_audio)
-    waveform, sample_rate = torchaudio.load(file_audio)
+    filepath = os.path.join(url, speaker_id, chapter_id, f"{fileid_audio}{ext_audio}")
+    full_path = os.path.join(root, filepath)
+    sample_rate = torchaudio.info(full_path).sample_rate
```

Comment:
Instead of fetch sample_rate here, why not hard code `16000` and assert it at `torchaudio.load`? LibriSpeech *is* expected to be 16kHz.